udfs/community/README.md

```diff
@@ -167,11 +167,11 @@ SELECT bqutil.fn.int(1.684)
 * [random_string](#random_stringlength-int64)
 * [random_value](#random_valuearr-any-type)
 * [studentt_cdf](#studentt_cdfx-float64-dof-float64)
-* [sure_cond](#sure_condvalue-string-cond-bool)
-* [sure_like](#sure_likevalue-string-like_pattern-string)
-* [sure_nonnull](#sure_nonnullvalue-any-type)
-* [sure_range](#sure_rangevalue-any-type)
-* [sure_values](#sure_valuesvalue-any-type-acceptable_value_array-any-type)
+* [sure_cond](#sure_cond)
+* [sure_like](#sure_like)
+* [sure_nonnull](#sure_nonnull)
+* [sure_range](#sure_range)
+* [sure_values](#sure_values)
```

Comment:
I think we want to keep the links as they were to function correctly

Reply:
https://github.com/GoogleCloudPlatform/bigquery-utils/pull/439

Reply:
ah I didn't realize my PR reverted the PR